src/App.jsx

```diff
@@ -51,6 +52,7 @@ const App = () => {
             <Route path='/chat' element={<JoinChat />} />
             <Route path='/chat/room' element={<ChatPage />} />
           </Route>
+          <Route path='/dashboard/seller' element={<SellerProductsPage />} />
```

Comment:
Please wrap this route in the `SellerRoutes` outlet to prevent non-Seller unauthorized access. The SellerRoutes outlet starts from this [Line 44 in App.jsx.](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/72ee91882f34b4cf35c8e6f114e81692b1c073ca/src/App.jsx#L44)